### DISCUSSION TITLE
Making this multi platform and support brew snap, appImage etc.

### DISCUSSION THREAD

**Discussion by jasonrgd** [Devolutions/UniGetUI#2158] [General]
Hi 
I see a lot of potential to make this a multi platform app store supporting various other platforms. 
For linux - supporting brew, apt and yum repositories, snap and flatpak 
For Mac - supporting brew and brew cask

Is there a plan for this ? I can help.

**Comment by marticliment** [comment]
There is an issue where this has been discussed

**Comment by Spacellary** [comment]
In my opinion – as a humble user – this at least seems out of scope for this project. Linux distros already have their own software stores and I'm unsure about WinUI3 and multi-platform...
